src/components/authentication/privateRoute.js

```diff
@@ -1,16 +1,15 @@
 import React, { Component } from 'react';
 import { Route, Redirect } from 'react-router-dom'
-
+import ApiService from '../customComponents/apiService';
 class PrivateRoute extends Component {
 
   render() {
     const logged = localStorage.getItem('loggedUser') ? true : false
```

Comment:
the ternary operator here is unnecessary: ` const logged = localStorage.getItem('loggedUser');`